### DISCUSSION TITLE
Windows zip download file - Defender shows as trojan Wacatac.B

### DISCUSSION THREAD

**Discussion by DjangoMaker** [mandiant/capa#2401] [General] [answered]
[download file](https://github.com/mandiant/capa/releases/download/v7.3.0/capa-v7.3.0-windows.zip) when unzipped is scanned by Windows Defender and shows it as Wacatac trojan. Attached screenshot. Is this normal?
![Screenshot 2024-09-24 140731](https://github.com/user-attachments/assets/0388b507-8205-4faf-aa58-e85340e49ffa)

**Comment by williballenthin** [comment, marked as answer]
We've had issues in the past with AV false positives due to the embedded rules that contain strings from malware. 

I don't believe there is malware embedded in the zip archive, and anyone is welcome to review the CI workflow that builds and uploads the file. 

Tomorrow I can submit a report to MS (assuming I can find the form).

**Reply by DjangoMaker** [reply]
Thank you.

**Comment by mr-tz** [comment]
Please also see the FAQ at https://github.com/mandiant/capa/blob/master/doc/faq.md.
